audformat/core/utils.py

```diff
@@ -851,47 +851,65 @@ def join_labels(
         joined labels
 
     Raises:
-        ValueError: if labels are of different type
-        ValueError: if label type is not ``list`` or ``dict``
+        ValueError: if labels are of different dtype
+            or not ``list`` or ``dict``
 
     Example:
         >>> join_labels([{'a': 0, 'b': 1}, {'b': 2, 'c': 2}])
         {'a': 0, 'b': 2, 'c': 2}
 
     """
     if len(labels) == 0:
-        return labels
+        return []
```

Comment:
This changes the type for an empty dict compared to the behavior we had before. Is this intended?

Reply:
You cannot pass an empty dict, only a list containing an empty dict, but in that case `labels` is not empty. I thought it's more obvious what happens when we change to `[]` here.

Reply:
Ah, yes, sorry forgot that you provide always a sequence of lists or dicts.